src/index.js

```diff
@@ -34,7 +35,10 @@ registerBlockType( 'nielslange/free-shipping-progress-bar', {
 	/**
 	 * @see ./edit.js
 	 */
-	edit: Edit,
+	// edit: Edit,
+	edit: withColors( { messageColor: 'color', progressColor: 'color' } )(
+		Edit
+	),
```

Comment:
Refrain from calling a HOC here. All Edit functionality should be exported as part of Edit component, so this HOC should be there.

Reply:
Thanks for your feedback. As requested, I moved the `withColors()` HOC from `index.js` to `edit.js`.